Multisig/UI/Transaction/TransactionDetailsViewController/Transaction Detail Cells/DetailRejectionInfoCell.swift

```diff
@@ -14,7 +14,13 @@ class DetailRejectionInfoCell: UITableViewCell, ExternalURLSource {
     @IBOutlet private weak var linkButton: UIButton!
     @IBOutlet private weak var helpLinkContinerView: UIView!
 
-    private(set) var url: URL? = App.configuration.help.payForCancellationURL
+    private(set) var url: URL? = {
+        var url = URL(string: "https://help.safe.global/en/articles/4738501-why-do-i-need-to-pay-for-cancelling-a-transaction")!
```

Comment:
I suggest to add //FIXME: everywhere to remove this boilerplate as soon as intercom migration is ready

Reply:
Done